lib/Driver/ToolChains.cpp

```diff
@@ -1471,6 +1512,124 @@ toolchains::Darwin::constructInvocation(const LinkJobAction &job,
   return II;
 }
 
+ToolChain::InvocationInfo
+toolchains::Windows::constructInvocation(const InterpretJobAction &job,
+                                         const JobContext &context) const {
+  InvocationInfo II = ToolChain::constructInvocation(job, context);
+
+  return II;
```

Comment:
Let's just take this out if we're not going to be doing anything with it right now.